constraints/BUCK

```diff
@@ -0,0 +1,97 @@
+constraint_setting(
+    name = "cpu",
+)
+
+constraint_value(
+    name = "aarch64",
+    constraint_setting = ":cpu",
+    visibility = ["PUBLIC"],
+)
+
+constraint_value(
+    name = "arm",
+    constraint_setting = ":cpu",
+    visibility = ["PUBLIC"],
+)
```

Comment:
do you still need this one?

Reply:
still needed here. to be fair this can use some improvement but havent fully thought out the best way to do it long term so we can circle back as we get more used to it